src/env/dev/terraform.tfvars

```diff
@@ -322,3 +324,12 @@ gpd_plan_sku_size = "S1"
 reporting_function_autoscale_minimum = 1
 reporting_function_autoscale_maximum = 3
 reporting_function_autoscale_default = 1
+
+# canone unico
+canoneunico_plan_kind     = "Linux"
```

Comment:
Since this value does not change among the environment you could set it as a default in varriable.tf file

Reply:
done 👍